relocation.bs

```diff
@@ -1592,11 +1591,10 @@ public:
     MyType(MyType);
 
     // Possible implementation:
-    static auto get_all(MyType tp)
+    auto get_all(this MyType self)
```

Comment:
```suggestion
    auto get_all(this MyType reloc)
```

Reply:
Not so sure about this. I aimed to show `get_all` as a non-decomposing function. I am afraid to lose the reader if I use a decomposing function before explaining what it is.

Reply:
Oh, I understand now, thanks!